src/Service/S3/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## NOT RELEASED
 
+### Fixed
+
+- According PHP documentation, format datetime with `ATOM` to provide a workaround for the incompatibility with `ISO8601` format.
```

Comment:
the fix should be described what is getting fixed in the library, not _how_  (as users of the library don't care about that).

So this should be ```Fix the format of the `objectLockRetainUntilDate` field in requests``` IMO